libs/utils/executor.py

```diff
@@ -272,13 +272,29 @@ def _setup_group(self, tc, group):
         # Configure each required attribute
         group.set(**tc['cgroups']['conf'][kind][name])
 
+    def _setup_files(self, tc):
+        errors = False
+        if 'files' not in tc:
+            logging.debug('%14s - Configuration not provided', 'Files')
+            return True
+        for name in tc['files']:
+            logging.info('%14s - Write value into file: \'%s\' -> \'%s\'',
+                         'Files', tc['files'][name], name)
+            try:
+                self.target.write_value(name, tc['files'][name], True)
+            except Exception:
+                logging.debug('%14s - Write value into file failed: \'%s\' -> \'%s\'',
+                         'Files', tc['files'][name], name)
+        return not errors
```

Comment:
I've changed it a little, we always check the write (to cause an exception to be thrown if it fails), and always catch the exception and log the failed write. If we are interested in a failure, then I re-raise the same exception. This gives me a nice log, plus the ability to completely fail if a required file isn't writeable for some reason.